packages/host/src/components/AppDrawer/AppDrawerIcon.tsx

```diff
@@ -16,7 +16,7 @@ export const AppDrawerIcon: React.FC = () => {
     return <MSupplyGuy size={drawer.isOpen ? 'large' : 'medium'} />;
 
   const style = drawer.isOpen
-    ? { paddingTop: 20, width: 64 }
+    ? { paddingTop: 20, width: 64, fill: theme.mixins.drawer?.iconColor }
```

Comment:
this was because the world food program icon should be white but was showing black when large and white when small